lib/generators/doorkeeper/openid_connect/templates/initializer.rb

```diff
@@ -24,9 +26,12 @@
     # redirect_to new_user_session_url
   end
 
-  subject do |resource_owner|
+  subject do |resource_owner, application|
     # Example implementation:
-    # resource_owner.key
+    # resource_owner.id
+
+    # or if you need pairwise subject identifier, implement like below:
+    # Digest::SHA256.hexdigest("#{resource_owner.id}#{URI.parse(application.redirect_uri).host}#{'your_secret_salt'}")
```

Comment:
> the Client MAY elect to provide its preferred identifier type using the subject_type parameter during Registration.

According to this description, we must implement features for [Open ID Connect Dynamic Client Registration](https://openid.net/specs/openid-connect-registration-1_0.html) to add ability for Client election of subject types, right?

If so, I think we should create another PR to cover multiple subject types since its implementation isn't trivial.

What's your opinion on this?  @toupeira